experiments/word2vec_evolution/analyze_theory_practice_bridges.py

```
@@ -0,0 +1,411 @@
+#!/usr/bin/env python3
+"""
+Analyze theory-practice bridges in the word2vec evolution experiment.
+Computes entropy maps and conveyance scores for paper-code relationships.
+"""
+
+import numpy as np
+import json
+from pathlib import Path
+from datetime import datetime
+import logging
+from typing import Dict, List, Tuple
```

Ruff (0.12.2)

`typing.Dict` is deprecated, use `dict` instead

(UP035)